bin/apalache-mc

```diff
@@ -56,6 +56,14 @@ sigterm() {
 
 trap sigterm SIGTERM SIGINT
 
+if [[ "$APALACHE_PKG_INFO" = true ]]
```

Comment:
I'm not sure if it make sense to document this functionality, and if so, where? CONTRIBUTING? Usage?

Basically, it seems to like something that's only useful when trying to debug a broken installation.

Reply:
Good point. Shall we report this, if the `--debug` is passed?

Reply:
Ah yeah, that's a better idea. I'll change this to check for that flag. Thanks!